backstop_data/engine_scripts/puppet/search/find-activities-delete-modal.js

```diff
@@ -6,6 +6,5 @@ module.exports = async (engine, scenario, vp) => {
   const page = await Page.build(engine, scenario, vp);
 
   await require('./find-activities')(engine, scenario, vp);
-  await engine.click('.crm-search-results span.crm-hover-button');
```

Comment:
and what if someone has civicase enabled? Shouldn't you cater for both possibilities?

Reply:
Since we are doing for the CiviCRM vanilla we probably can add these information in readme. We have two options.
1) Can tell the user to disable Civicase components in readme
2) Or Specifically ask user to enable  Civicase components and we will uncomment the above line.

Reply:
i'd rather have that the script is able to click "delete" regardless of where it is

Reply:
Done!